src/dbms/replication_client.cpp

```diff
@@ -10,25 +10,32 @@
 // licenses/APL.txt.
 
 #include "dbms/replication_client.hpp"
+#include "replication/replication_client.hpp"
 
 namespace memgraph::dbms {
 
 void StartReplicaClient(DbmsHandler &dbms_handler, replication::ReplicationClient &client) {
   // No client error, start instance level client
   auto const &endpoint = client.rpc_client_.Endpoint();
   spdlog::trace("Replication client started at: {}:{}", endpoint.address, endpoint.port);
-  client.StartFrequentCheck([&dbms_handler](std::string_view name) {
-    // Working connection, check if any database has been left behind
-    dbms_handler.ForEach([name](dbms::Database *db) {
+  client.StartFrequentCheck([&client, &dbms_handler](std::string_view name, bool reconnect) {
```

Comment:
`client.StartFrequentCheck([&client`
Can't we change the call arguments of the lambda to receive the client, rather than capture by reference the client.